examples/widgets/bottom_sheet.dart

```diff
@@ -0,0 +1,64 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
```

Comment:
Just to be clear, I do think it's useful to have the feature used in a demo. The feature will bit rot if it's not used frequently. (This was the reasoning behind the stocks app and the fitness app - they were supposed to be apps that team members would use in the real world and so see real issues.)